README.md

```diff
@@ -67,14 +65,14 @@ Oh My Fish includes a small utility `omf` to fetch and install new packages and
 
 ## `omf update`
 
-Update framework and installed packages.
+Update the framework and installed packages.
 
-## `omf install` _`[package|url ...]`_
+## `omf install` _`[<package|url> ...]`_
```

Comment:
`####` instead of `##` is going to look better.

`omf install [<name>|<url>]` this is how we specify it on `omf help`. As the latest update was made on `omf help`, we better copy the ideas from there.